ROADMAP.md

```diff
@@ -0,0 +1,34 @@
+Roadmap
+---
+This represents the bigger ticket items we have planned for K3s. There are many more tactical enhancements and fixes that can be found by perusing our [GitHub milestones](https://github.com/rancher/k3s/milestones).
+
+
+v1.19 - End of August 2020
+---
+- Add support for config file based configuration
+- Replace experimental dqlite support with etcd
+- Support ability to customize baked-in Helm charts
+- Improve and refactor SELinux support
+- Improve CentOS and RHEL 7 and 8 support
+
+
+v1.20 - December 2020
+---
+- Disentangle code and docs from Rancher Labs
+- Support a 2-node hot/cold HA model by leveraging etcd's learner node feature
+- Introduce `k3s build` feature for easier local image building
+- Reduce patch set k3s requires for Kubernetes
+- Migrate to using upstream’s kubelet certificate rotation rather than our own custom logic
+- Migrate to using the kubelet config file for configuring kubelets rather than command line args
+- Upgrade to Traefik 2.x for ingress
+- Use "staging" location for binaries as part of upgrades
+
+
+Backlog
+---
+- Windows support
+- Introduce "Cloud Provider" builds that have CSI and CCM baked-in for each provider
```

Comment:
Nit: Consider avoiding phrases like "baked-in" that may not make sense to folks who speak English as a second language.

e.g. baked-in -> included

Additionally, think about this throughout. For example, perhaps we need to rethink "bigger ticket" on line 3?